packages/html_of_jsx/html_of_jsx.0.0.3/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "Render HTML with JSX"
+description:
+  "html_of_jsx is a JSX transformation that allows you to write HTML declaratively."
+maintainer: ["David Sancho <dsnxmoreno@gmail.com>"]
+authors: ["David Sancho <dsnxmoreno@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/davesnx/html_of_jsx"
+bug-reports: "https://github.com/davesnx/html_of_jsx/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml" {>= "5.0.0"}
+  "reason" {>= "3.10.0"}
+  "ppxlib" {> "0.23.0"}
+  "rope" {>= "0.6.3"}
+  "alcotest" {with-test}
+  "benchmark" {with-test}
+  "odoc" {with-doc}
+  "ocamlformat" {= "0.26.1" & with-dev-setup}
```

Comment:
```suggestion
  "ocamlformat" {= "0.26.1" & (with-dev-setup |with-test)}
```

Or something along those lines bc it seems the tests still depend on it:

```
#=== ERROR while compiling html_of_jsx.0.0.3 ==================================#
# context              2.2.0~beta3~dev | linux/x86_64 | ocaml-base-compiler.5.0.0 | pinned(https://github.com/davesnx/html_of_jsx/releases/download/0.0.3/html_of_jsx-0.0.3.tbz)
# path                 ~/.opam/5.0/.opam-switch/build/html_of_jsx.0.0.3
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p html_of_jsx -j 255 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/html_of_jsx-7-f5f69c.env
# output-file          ~/.opam/log/html_of_jsx-7-f5f69c.out
### output ###
# File "test/cram/dune", line 3, characters 53-71:
# 3 |  (deps ppx.sh %{bin:jsx-ppx-standalone} %{bin:refmt} %{bin:ocamlformat}))
#                                                          ^^^^^^^^^^^^^^^^^^
# Error: Program ocamlformat not found in the tree or in PATH
#  (context: default)
```

Reply:
This type of tests is very brittle, I would strongly suggest to have formatting tests run separately in a manual step.